packages/jest-cli/src/TestRunner.js

```diff
@@ -16,9 +16,14 @@ import type {
 import type {GlobalConfig, ReporterConfig} from 'types/Config';
 import type {Context} from 'types/Context';
 import type {Reporter, Test} from 'types/TestRunner';
-import type {PathPattern} from './SearchSource';
+import type {PathPatternObject} from './SearchSource';
 
 import {formatExecError} from 'jest-message-util';
+import {
+  addResult,
+  buildFailureTestResult,
+  makeEmptyTestResult,
+} from './test_result_utils';
```

Comment:
@cpojer won't be happy haha. Maybe we can name it `test_result_helpers`. I like that we're reusing `makeEmptyTestResult` – how about renaming it to `makeEmptyAggregatedResult`, just like we call its type?